### PR TITLE
post release 1.5.0

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

**Comment by tjhei** [comment]
This is the draft for the release notes, please take a look/modify but don't merge yet.

File: doc/release-tasklist [hunk 1]
```diff
@@ -71,6 +71,51 @@
...
+
+This release includes the following changes:
+
+- New: Choice between different formulations for the governing equations
+  including Boussinesq and anelastic liquid approximation
+- New: Melt transport (two-phase flow)
+- New: Nondimensional material model for incompressible (using the Boussinesq
+  approximation) and compressible computations (with ALA or TALA) for
+  nondimensionalized problems. This can be used for benchmark problems like
+  Blankenbach, King, etc..
```
Comment (bangerth) [inline doc/release-tasklist]: be consistent with periods at the end of each bullet points (either with or without; the last one always with).

File: doc/release-tasklist [hunk 2]
```diff
@@ -71,6 +71,51 @@
...
+- New: optional DG method for temperature/composition
```
Comment (bangerth) [inline doc/release-tasklist]: capitalize "optional". Be consistent everywhere.

File: doc/release-tasklist [hunk 3]
```diff
@@ -71,6 +71,51 @@
...
+- Adiabatic conditions: rework, now includes a reference density profile
+- New cookbooks: continental extension, finite strain
+- New: viscoplastic material model
+- Material model interface cleanup
+- Particles: new generators, ghost exchange, performance improvements,
```
Comment (bangerth) [inline doc/release-tasklist]: I would move this up: There are many many improvements, it is vastly better than it was before.

File: doc/release-tasklist [hunk 4]
```diff
@@ -71,6 +71,52 @@
...
+  including Boussinesq and anelastic liquid approximation.
+- New: Melt transport (two-phase flow).
+- Particles: new generators, ghost exchange, performance improvements,
+  interpolation to fields.
+- New: Nondimensional material model for incompressible (using the Boussinesq
+  approximation) and compressible computations (with ALA or TALA) for
+  nondimensionalized problems. This can be used for benchmark problems like
+  Blankenbach, King, etc..
+- New: Optional DG method for temperature/composition.
+- Adiabatic conditions: rework, now includes a reference density profile
```
Comment (bangerth) [inline doc/release-tasklist]: period at the end like in all other items

File: doc/release-tasklist [hunk 5]
```diff
@@ -71,6 +71,52 @@
...
+- Free surface: overhaul.
+- New cookbooks: continental extension, finite strain.
```
Comment (bangerth) [inline doc/release-tasklist]: burnman; did we not have others as well?

Comment (bangerth) [inline doc/release-tasklist]: inner core?

**Pull request merged**